bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -750,4 +750,8 @@ spec:
     name: cert-manager-controller
   - image: quay.io/jetstack/cert-manager-acmesolver:v1.11.4
     name: cert-manager-acmesolver
+  skips:
+  - cert-manager-operator.v1.11.1
+  - cert-manager-operator.v1.10.3
```

Comment:
Do we oversee any problems if this goes out and we never do a z-stream for v1.10 (i.e. 1.10.3 is never shipped)?

Reply:
Should not be a problem.
```
➜  1.11.4 IMAGE=quay.io/thejasn/cert-manager-operator-bundle:v1.11.4 && echo "FROM ${IMAGE}" > Dockerfile && DOCKER_BUILDKIT=1 podman build -o extracted .
STEP 1/1: FROM quay.io/thejasn/cert-manager-operator-bundle:v1.11.4
COMMIT
--> b19d33fb3ffb
b19d33fb3ffb7f8f1c1f97109a5272a50e2cbb84937da1ac16b93cd0132ee06c
➜  1.11.4 cat extracted/manifests/cert-manager-operator.clusterserviceversion.yaml | grep skips -a10
    name: Red Hat
  relatedImages:
  - image: quay.io/jetstack/cert-manager-webhook:v1.11.4
    name: cert-manager-webhook
  - image: quay.io/jetstack/cert-manager-cainjector:v1.11.4
    name: cert-manager-ca-injector
  - image: quay.io/jetstack/cert-manager-controller:v1.11.4
    name: cert-manager-controller
  - image: quay.io/jetstack/cert-manager-acmesolver:v1.11.4
    name: cert-manager-acmesolver
  skips:
  - cert-manager-operator.v1.11.2
  - cert-manager-operator.v1.11.1
  - cert-manager-operator.v1.10.6
  - cert-manager-operator.v1.10.2
  version: 1.11.4
➜  1.11.4 oc get csv -n cert-manager-operator
NAME                            DISPLAY                                       VERSION   REPLACES   PHASE
cert-manager-operator.v1.11.1   cert-manager Operator for Red Hat OpenShift   1.11.1               Succeeded
➜  1.11.4 oc get subscriptions -n cert-manager-operator openshift-cert-manager-operator -o=jsonpath={.status.installedCSV}
cert-manager-operator.v1.11.1%
➜  1.11.4 oc get subscriptions -n cert-manager-operator openshift-cert-manager-operator -o=jsonpath={.status.state}
UpgradePending%
➜  1.11.4 oc get installplan -n cert-manager-operator
NAME            CSV                             APPROVAL   APPROVED
install-dmzb5   cert-manager-operator.v1.11.4   Manual     false
install-msbk8   cert-manager-operator.v1.11.1   Manual     true
```

Reply:
Thanks for confirming!